scripts/release/asana-update-tasks.js

```diff
@@ -27,35 +22,39 @@ const setupAsana = () => {
 }
 
 const asanaUpdateTasks = async () => {
+    let asanaOutput
+    try {
+        asanaOutput = JSON.parse(asanaOutputRaw || '')
+    } catch (e) {
+        throw new Error('Unable to parse Asana output JSON')
+    }
+
     setupAsana()
 
     const platformEntries = Object.entries(asanaOutput)
     for (const [platformName, platformObj] of platformEntries) {
-        // If either are absent we haven't implemented the automation for that platform
+        // If we're missing required data, either we haven't implemented automation for that platform yet,
+        // or something went wrong in a previous job
         if (!platformObj.taskGid || !prUrls[platformName]) continue
 
-        // Get the task
         const { html_notes: notes } = await asana.tasks.getTask(platformObj.taskGid, { opt_fields: 'html_notes' })
 
+        const prLink = getLink(prUrls[platformName], `${platformObj.displayName} PR`)
         /** @type {[[RegExp, string]]} */
         const taskDescriptionSubstitutions = [
-            [/\[\[pr_url]]/, getLink(prUrls[platformName], platformObj.displayName + ' PR')]
+            [/\[\[pr_url]]/, prLink]
         ]
 
+        let extraContent = ''
         if (platformName === 'bsk') {
             // On the BSK task we also substitute the ios and macos placeholders
-            const markup =
-                `${wrapInLi(getLink(prUrls.ios, 'iOS PR'))}${wrapInLi(getLink(prUrls.macos, 'macOS PR'))}`
-            taskDescriptionSubstitutions.push(
-                [/\[\[extra_content]]/, markup]
```

Comment:
Let me know if I've made a mistake here, but it seemed strange that here we didn't replace the wrapping `<li>`, but below we did -- surely we'd want to replace it both times? (either to replace it with the two bullet points created above, or to remove it completely)

Reply:
Mhm, not 100% sure, but we'll see when it runs.